tests/target/expr.rs

```diff
@@ -290,3 +290,16 @@ fn complex_if_else() {
         yo();
     }
 }
+
+fn issue1106() {
+    if let hir::ItemEnum(ref enum_def, ref generics) =
+           self.ast_map
```

Comment:
ditto here

Also, the method chain looks like it should fit on one line rather than three